Sources/graphql-swift/codegen/jsonUtils.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+public protocol InitializableFromJSON {
+    init(_ json: [String: Any])
+}
+
+public extension Dictionary where Key == String {
+    func val(_ key: String) -> [String: Any]? {
+        return self[key] as? [String:Any]
```

Comment:
Nit: 
```suggestion
        return self[key] as? [String: Any]
```

This is just a code style though 😸  You’re using it in the method signature though.